Fix #145 to datatable v2.0.3

alexsilva opened the pull request:
I was already using the lib in version 1.x of the datatable, but when migrating to version 2.x, I got the failure described in issue [#145](https://github.com/gyrocode/jquery-datatables-checkboxes/issues/145) I made the corrections based on version 2.0.3 of the datatable (the internal apis are private were copied to source).

alexsilva committed:
Restoring the basic functionality of rendering checkboxes.

alexsilva committed:
Adjustments to execute node(s) for cell or cells.

alexsilva committed:
Correction of information display for selected items.

alexsilva committed:
Version validation to run local implementation of '_fnColumnOptions'.

alexsilva committed:
Added [select/deselect].dtcheckboxes events.

alexsilva committed:
update version 1.3.0.

alexsilva committed:
Compiled version v1.3.0

alexsilva committed:
Ignore idea.

DPComp commented:
how can I download your corrections?

alexsilva commented:
> how can I download your corrections?

I generated a "compiled" version v1.3.0 with the [Source](https://github.com/alexsilva/jquery-datatables-checkboxes/blob/master/js/dataTables.checkboxes.min.js)

DPComp commented:
> > how can I download your corrections?
> 
> I generated a "compiled" version v1.3.0 with the [Source](https://github.com/alexsilva/jquery-datatables-checkboxes/blob/master/js/dataTables.checkboxes.min.js)

thanks
the css file, remains de same right?

DPComp commented:
> > how can I download your corrections?
> 
> I generated a "compiled" version v1.3.0 with the [Source](https://github.com/alexsilva/jquery-datatables-checkboxes/blob/master/js/dataTables.checkboxes.min.js)

Alex, I updated the js files + datatables 2.0.3
It stoped the failure, but it also stoped working... it's not showing the checkboxes anymore

example

  columnDefs: [{
    targets: 0,
    orderable: false,
    checkboxes: { selectRow: true },
  },

this used to show a column with checkbox in the 1st row, and in the datatables 2.0.3 + your correction v1.3.0, it's not

alexsilva committed:
Fix invalidate column data.

alexsilva committed:
Fix grunt rules.

alexsilva committed:
Source code with checkbox corrections (data invalidation).

alexsilva commented:
> > > how can I download your corrections?
> > 
> > 
> > I generated a "compiled" version v1.3.0 with the [Source](https://github.com/alexsilva/jquery-datatables-checkboxes/blob/master/js/dataTables.checkboxes.min.js)
> 
> Alex, I updated the js files + datatables 2.0.3 It stoped the failure, but it also stoped working... it's not showing the checkboxes anymore
> 
> example
> 
> columnDefs: [{ targets: 0, orderable: false, checkboxes: { selectRow: true }, },
> 
> this used to show a column with checkbox in the 1st row, and in the datatables 2.0.3 + your correction v1.3.0, it's not

I updated the source with a [fix](https://github.com/gyrocode/jquery-datatables-checkboxes/pull/147/files#diff-69e3f37d4e6acbb448981cc3d7611f1f72a19100e769da7a96cc120f97ed8679L160) for this described problem.

I kept the version but generated a new min code.
